opentracing-api/src/main/java/io/opentracing/tag/Tags.java

```diff
@@ -63,7 +63,7 @@ private Tags() {
     /**
      * PEER_HOST_IPV4 records IPv4 host address of the peer.
      */
-    public static final IntTag PEER_HOST_IPV4 = new IntTag("peer.ipv4");
+    public static final IntOrStringTag PEER_HOST_IPV4 = new IntOrStringTag("peer.ipv4");
```

Comment:
By doing this there is not a way how to deprecate `int` part. Spec says that it's `string` https://github.com/opentracing/specification/blob/master/semantic_conventions.md.